release_images.yml

```diff
@@ -4,31 +4,41 @@ release_images:
     framework: "tensorflow"
     version: "1.15.3"
     training:
-      device_types: ["cpu", "gpu"]
-      python_versions: ["py37", "py36", "py27"]
+      device_types: ["gpu"]
+      python_versions: ["py37", "py27"]
       os_version: "ubuntu18.04"
       cuda_version: "cu100"
       example: False        # [Default: False] Set to True to denote that this image is an Example image
       disable_sm_tag: False  # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
                             # to images being published.
-      force_release: False  # [Default: False] Set to True to force images to be published even if the same image
+      force_release: True  # [Default: False] Set to True to force images to be published even if the same image
                             # has already been published. Re-released image will have minor version incremented by 1.
-    inference:
-      device_types: ["cpu", "gpu"]
-      python_versions: ["py36", "py27"]
-      os_version: "ubuntu18.04"
-      cuda_version: "cu100"
-      example: False
-      disable_sm_tag: False
-      force_release: False
   2:
     framework: "tensorflow"
-    version: "1.15.3"
-    training:
-      device_types: ["gpu"]
-      python_versions: ["py37", "py36"]
-      os_version: "ubuntu18.04"
-      cuda_version: "cu100"
-      example: True
-      disable_sm_tag: False  # [Default: False] This option is not used by Example images
-      force_release: False
+      version: "2.2.0"
+      training:
+        device_types: ["cpu", "gpu"]
+        python_versions: ["py37"]
+        os_version: "ubuntu18.04"
+        cuda_version: "cu101"
+        example: False        # [Default: False] Set to True to denote that this image is an Example image
+        disable_sm_tag: False  # [Default: False] Set to True to prevent SageMaker Abbreviated Tags from being attached
+        # to images being published.
+        force_release: False  # [Default: False] Set to True to force images to be published even if the same image
+        # has already been published. Re-released image will have minor version incremented by 1.
+      inference:
+        device_types: ["cpu", "gpu"]
+        python_versions: ["py37"]
+        os_version: "ubuntu18.04"
+        cuda_version: "cu102"
+  3:
```

Comment:
Add the `disable_sm_tags: False` and `force_release: False` options here for the TF 2.2 inference image.